Documentation/concepts/ipam/cluster-pool.rst

```diff
@@ -6,13 +6,24 @@
 
 .. _ipam_crd_cluster_pool:
 
-########################
-Cilium Cluster-pool IPAM
-########################
+#######################
+Cluster Scope (Default)
+#######################
 
-Cilium Cluster-pool IPAM is based on Kubernetes host-scope IPAM, for more info
-see :ref:`k8s_hostscope`. The functionality is the same but the ``PodCIDRs`` are
-allocated and managed entirely by Cilium Operator.
+The Cluster scope IPAM mode is almost identical to the :ref:`k8s_hostscope`
+mode. It also works based on a PodCIDR assigned to each node which is then used
```

Comment:
Should we start by a single-sentence description of what this is and only mention that it's similar to hostscope later on in the paragraph? Starting with a comparison to hostscope requires the reader to already understand hostscope.